src/main/java/org/opensearch/ad/ml/ThresholdingResult.java

```diff
@@ -58,8 +95,9 @@ public ThresholdingResult(
         boolean inHighScoreRegion,
         int relativeIndex,
         double[] currentTimeAttribution,
```

Comment:
nit: rename this to `relevantAttribution` also?

Reply:
renamed